aredis/commands/sorted_set.py

```diff
@@ -75,6 +77,31 @@ class SortedSetCommandMixin:
             pieces.append(pair[0])
         return await self.execute_command('ZADD', name, *pieces)
 
+    async def zaddoption(self, name, option=None, *args, **kwargs):
```

Comment:
Since the implementation is similar to that of `zadd`, maybe we can just add an optional parameter `option` to `zadd`, that will be more friendly to users

Reply:
I thought a bit about it, but I'm not sure it can be done safely in a backwards compatible manner.

If we add it before *args, it eats into *args so existing zadd calls stop working.
If we add it as a **kwarg, it is POSSIBLE somebody has a key named "NX" or "XX" (unlikely, but possible).